scan/lib/scan/detect_values.rb

```diff
@@ -62,26 +72,38 @@ def self.default_device_ios
       found = sims.detect { |d| d.name == "iPhone 5s" }
       found ||= sims.first # anything is better than nothing
 
-      Scan.device = found
-
-      UI.user_error!("No simulators found") unless Scan.device
+      if found
+        Scan.devices = [found]
+      else
+        UI.user_error!("No simulators found on local machine")
+      end
     end
 
     def self.default_device_tvos
-      config = Scan.config
-
-      if config[:device] # make sure it actually exists
-        device = config[:device].to_s.strip.tr('()', '') # Remove parenthesis
-
-        found = FastlaneCore::SimulatorTV.all.find do |d|
-          (d.name + " " + d.tvos_version).include? device
+      devices = Scan.config[:devices] || [Scan.config[:device]] # important to use Array(nil) for when the value is nil
```

Comment:
`[Scan.config[:device]]` does not match the comment about needing to use `Array(...)`